dev/com.ibm.ws.componenttest.2.0/bnd.bnd

```diff
@@ -28,7 +28,9 @@ Export-Package: \
   org.hamcrest.*;version=1.0.16
   
 Private-Package: \
-  componenttest.serverinfo.impl
+  componenttest.serverinfo.impl,\
+  javax.json.*,\
```

Comment:
This should be jakarta.json?

Reply:
I did try using jakarta.json but that created a classpath issue because fattest.simplicity already uses javax.json. Because these are standalone test bundles, it seemed easier to just use javax.json.